docs: Adding 'Integrating with Kubernetes' user guide (#2250)
codenvy/che-docs: modules/end-user-guide/pages/automatic-token-injection.adoc

:navtitle: Automatic {orch-name} token injection
:keywords: user-guide, token, injection
:page-aliases: .:automatic-token-injection, overview:automatic-token-injection

[id="automatic-token-injection_{context}"]
= Automatic {orch-name} token injection

> This section describes how to use the {orch-name} user token that is automatically injected into workspace containers.

Reviewer: It would be cool if this sentence could provide information on what you can do with the user token.

Author: fair point, will rework it

.Procedure

. Open the {prod-short} dashboard and start a workspace.

> . Once the workspace is started open a terminal in the container that contains {prod-short} CLI.

Reviewer: ```suggestion
. Once the workspace is started, open a terminal in the container that contains the {prod-short} CLI.
```

> . Execute {prod-short} CLI commands which allow you to run commands against {orch-name} cluster. CLI can be used for deploying applications, inspecting and managing cluster resources, and viewing logs. {orch-name} user token will be used during the execution of the commands. 

Reviewer: ```suggestion
. Execute {prod-short} CLI commands to run commands against {orch-name} cluster. CLI can be used for deploying applications, inspecting and managing cluster resources, and viewing logs. The {orch-name} user token will be used during the execution of the commands. 
```

+
image::integration/token-injection.png[Token Injection in IDE,link="../_images/integration/token-injection.png"]

> WARNING: The automatic token injection functionality currently works only on the OpenShift infrastructure.

Reviewer: "function" is easier than "functionality" and provides the same content.

Author: I decided to drop `functionality` wording since it is redundant

:navtitle: Automatic {orch-name} token injection
:keywords: user-guide, token, injection
:page-aliases: .:automatic-token-injection, overview:automatic-token-injection

[id="automatic-token-injection_{context}"]
= Automatic {orch-name} token injection

This section describes how to use {orch-name} user token that is automatically injected in workspace containers.

.Procedure

> . Open the {prod-short} dashboard and start a workspace.

Reviewer 2: ```suggestion
1. Open the {prod-short} dashboard and start a workspace.
```

> . Once workspace is started open a terminal in the container that contains {prod-short} CLI.

Reviewer 2: ```suggestion
1. After the workspace is started, open a terminal in the container that contains {prod-short} CLI.
```

Reviewer 3: @swatikarmarkar The syntax was correct.

> . Execute {prod-short} CLI commands. {orch-name} user token will be used during the commands execution.

Reviewer 2: ```suggestion
1. Execute {prod-short} CLI commands. {orch-name} user token will be used during the execution of the commands.
```
